public/pages/Reindex/utils/helper.ts

```diff
@@ -55,3 +55,15 @@ export const checkDuplicate = (
   }
   return null;
 };
+
+export const filterOverlaps = (
+  list: EuiComboBoxOptionOption<IndexSelectItem>[],
+  excludeList?: EuiComboBoxOptionOption<IndexSelectItem>[]
```

Comment:
Can we make excludeList a required parameter? If it's not provided, there is no need to call this function